src/util/XDRCereal.cpp

```diff
@@ -0,0 +1,39 @@
+
+#include "util/XDRCereal.h"
+
+void
+cereal_override(cereal::JSONOutputArchive& ar, const stellar::PublicKey& s,
+                const char* field)
+{
+    xdr::archive(ar, stellar::KeyUtils::toStrKey<stellar::PublicKey>(s), field);
+}
```

Comment:
Nitpick: missing whitespace here and between lines 33-34. Code is written once, but (hopefully) maintained for a long time. So try to make life easy on the maintainer when possible.

Reply:
Makes sense! Done!